server.js

```diff
@@ -0,0 +1,46 @@
+const express = require('express')
+const app = express()
+
+const { sequelize, testConnection } = require('./utils/connect')
+
+app.set('view engine', 'ejs')
+// required for forms
+app.use(express.urlencoded({extended: true}))
+// for JSON request
+app.use(express.json())
+
+testConnection()
+sequelize.sync({force: true}).then(()=>{
+    console.log('db is ready')
+})
+
+// STATIC example (/test/tt.html) navigate all pages in directory
+// app.use(express.static("public"))
+
+// add use logger to add middleware to every route
+// app.use(logger)
+
+// get/post/put/delete/patch, tend not to use next param on get post, can have as many middlewares as needed
+// app.get("/", logger, logger, (req, res))...
+app.get('/', logger, (req, res) => {
+    // console.log('here')
+    // res.sendStatus(500)
+    // res.status(200).send('HI')
+    // res.status(200).json({message: 'error'})
+    // res.download('server.js')
+    // res.json({message: 'error'})
+    res.render('index', { text: 'world'})
+})
+
+const userRouter = require('./routes/users')
+
+// link route to path
+app.use('/users', userRouter)
+
+// all middleware takes req res next
+function logger(req, res, next) {
```

Comment:
suggestion: as a bit of tidy up - perhaps make a middleware folder and have this there `middleware/logger.js`

You can also add it globally before all your routes so you don't need to do this above
```js
app.get('/', logger, (req, res) => {
```

instead just do

```js
const loggerMiddleware = require('./middleware/logger');

...

app.use(loggerMiddleware);

```